### DISCUSSION TITLE
Slogan for Faker

### DISCUSSION THREAD

**Discussion by Shinigami92** [faker-js/faker#1263] [Ideas]
(This idea originally was raised by @xDivisionByZerox)

Currently in our docs we have a slogan saying:

> Generate massive amounts of fake (but realistic) data for testing and development.

<details>
<summary>Screenshot</summary>

<img width="930" alt="image" src="https://user-images.githubusercontent.com/7195563/184326895-96150c21-230c-4995-ab2b-6fc57468097c.png">

</details>

We thought about a more catchy slogan:

> Fake it 'til you make it!

So while the current slogan is still our "What is Faker" sentence, the new slogan would be more of a battle cry.

**Comment by ST-DDT** [comment]
`Fake it, don't make it.`

**Comment by ST-DDT** [comment]
`Fake it now, for an early wow` 
(not sure whether I got the right spelling)

**Comment by xDivisionByZerox** [comment]
`Fake it 'til you make it (to production)!`

A little more what I had in mind. This emphasizes on us being a library for development.

**Reply by JessicaSachs** [reply]
I really love this as a slogan on a T-Shirt! 😸

**Comment by HonzaMac** [comment]
Randomize your fixtures for maintainable tests

**Reply by Shinigami92** [reply]
Not bad, but feels more like a description as a slogan
We might want to add this to docs/readme anyways

**Comment by HonzaMac** [comment]
'Make data like Faker'

Reminds me [Fakir](https://en.wikipedia.org/wiki/Fakir?wprov=sfti1).

**Comment by woldaker** [comment]
Faker -
_Convert use cases into test cases.  Not users into testers._
